src/validator-map.js

```diff
@@ -53,8 +53,15 @@ VALIDATOR_MAP[DATA_TYPES.TIME] = Utils.buildRegexCheck('isTime');
 // VALIDATOR_MAP[DATA_TYPES.TIME] = Utils.whichFormatTime;
 
 // numbers:
-// 1, 2, 3
-VALIDATOR_MAP[DATA_TYPES.INT] = Utils.buildRegexCheck('isInt');
+// 1, 2, 3, +40, 15,121
+VALIDATOR_MAP[DATA_TYPES.INT] = function isInt(value) {
+  if (Utils.buildRegexCheck('isInt')(value)) {
```

Comment:
`buildRegexCheck` should not be called in every check.

You should define intRegex outside the `isInt` function:
```
const intRegexCheck = Utils.buildRegexCheck('isInt');
function isInt(value) {
   if (intRegexCheck(value)) {
    ....
   }
}
```

Reply:
updated